reana_server/reana_admin/check_workflows.py

```diff
@@ -30,34 +32,36 @@
 from reana_server.reana_admin.consumer import CollectingConsumer
 
 
-class WorkflowCheckFailed(Exception):
-    """Error when workflow doesn't pass some check."""
+class CheckFailed(Exception):
+    """Error when check fails."""
 
 
-WorkflowCheckResult = Tuple[Workflow, List[WorkflowCheckFailed]]
+CheckSource = namedtuple("CheckSource", "id name user status")
+CheckResult = Tuple[CheckSource, List[CheckFailed]]
```

Comment:
Because I also want types, I replaced it with `dataclass` ([python docs](https://docs.python.org/3.8/library/dataclasses.html), [StackOverflow](https://stackoverflow.com/questions/34269772/type-hints-in-namedtuple)). Learned something new :)